### PR TITLE
Update ACF Pro download endpoint

### PR DESCRIPTION
**Pull request by nlemoine**
Hi @pivvenci 

ACF download URL recently changed.

### CODE REVIEW

File: build.php [hunk 1]
```diff
@@ -42,7 +42,7 @@ function createPackage($tag, $alias = null, $type = "wpackagist-plugin", $repoVe
         "keywords" => "acf, advanced, custom, field, fields, form, repeater, content",
         "dist" => (object)[
             "type" => "zip",
-            "url" => "https://connect.advancedcustomfields.com/index.php?p=pro&a=download&t={$tag}"
+            "url" => "https://connect.advancedcustomfields.com//v2/plugins/download?p=pro&t={$tag}"
```
Comment (schliflo) [inline build.php]: double slash here ...`com//v2`... should probably be removed.

Comment (nlemoine) [reply build.php]: Whoops, sorry, this is fixed.

**Comment by nlemoine** [comment]
@schliflo I fixed the typo ;)

**Comment by Qrious** [comment]
I'll update this, this also requires an update of our installer. I'll release the new version as well

**Comment by nlemoine** [comment]
I didn't notice download URL was also somewhere in installer. Thanks!

**Comment by Qrious** [comment]
For now ACF's API is still backwards compatible :). I'm still giving this some thought, to make sure we have the best upgrade path. I think it is important that I update the installer in a backwards compatible manner. 

I'll look further at this later today.

**Comment by Qrious** [comment]
I've released a new version (2.3) of the installer. However I think we should only use the new download link for releases >= 5.8.8, and then also require ^2.3 of the installer for these newer versions. Otherwise current versions break with the old installers.

**Comment by Qrious** [comment]
I've added these changes. What do you guys think, does this break compatibility with any of your projects (I don't think it should)?

**Comment by nlemoine** [comment]
> For now ACF's API is still backwards compatible :)

I just checked, it is but I actually made this PR because I had a 404 on the old download link a few days ago. 

> What do you think, does this break compatibility with any of your projects (I don't think it should)?

I think you shouldn't bother to target different URLs, the new endpoint works fine with every ACF version ;)

**Comment by Qrious** [comment]
That does however break backwards compatibility with people that have the old versions in their lock file, together with the old installers, so i've decided to support the new URL's for the latest release and upwards (as does ACF-PRO when used without composer).

**Comment by nlemoine** [comment]
I didn't catch the installer issue. Great job. Thanks for merging this!

**Pull request merged**